i18n/locales/de/common.json

```diff
@@ -229,5 +229,10 @@
   "{{count}} of the delegate names selected for voting was already voted for for:": "{{count}} der ausgewählten Delegierten-Namen zum Wählen sind bereits gewählt:",
   "{{count}} of the delegate names selected for voting was already voted for for:_plural": "{{count}} der ausgewählten Delegierten-Namen zum Wählen sind bereits gewählt:",
   "{{count}} of the provided delegate names could not be resolved:": "{{count}} der eingegebenen Delegierten-Namen konnten nicht gefunden werden:",
-  "{{count}} of the provided delegate names could not be resolved:_plural": "{{count}} der eingegebenen Delegierten-Namen konnten nicht gefunden werden:"
+  "{{count}} of the provided delegate names could not be resolved:_plural": "{{count}} der eingegebenen Delegierten-Namen konnten nicht gefunden werden:",
+  "[Today], hh:mm A": "[Heute], hh:mm A",
+  "[Tomorrow], hh:mm A": "[Morgen], hh:mm A",
+  "[Yesterday], hh:mm A": "[Gestern], hh:mm A",
+  "{{days}}d ago": "Vor {{days}}d",
+  "{{hours}}h ago": "Vor {{hours}}h"
```

Comment:
These strings are missing from `i18n/locales/en/common.json`. Looks like they are not automatically generated, which will be a problem.

Reply:
Oh actually these were already on the `en/common.json`, I just added  to `de/common.json`,  to be able to test.
https://github.com/LiskHQ/lisk-hub/blob/6d4512c60680992a49c079a5ded1425855856396/i18n/locales/en/common.json#L521-L531

Not exactly same order, as I just added them to the end of file.

Reply:
Ah, got it now 👍